app-creator/go.sum

```diff
@@ -0,0 +1,351 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
Is this file necessary to check into the repo? Is it generated automatically?

Reply:
> Ensure your go.sum file is committed along with your go.mod file. See FAQ below for more details and rationale.
https://github.com/golang/go/wiki/Modules#releasing-modules-all-versions

The checksum helps ensure that whoever is running the project can have the exact version of the dependencies.